doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -119,22 +119,19 @@ Detailed info about neon's features and functionality can be found in the
 `neon docs`_. This section covers installing neon on an existing 
 system that already has an ``ngraph_dist`` installed. 
 
-.. important:: The numbered instructions below pick up from where 
-   the :doc:`install` instructions left off, and they presume that your system 
-   already has the ngraph library installed installed at ``$HOME/ngraph_dist`` 
-   as the default location. If the |nGl| code has not yet been installed to 
-   your system, you can follow the instructions on the `ngraph-neon python README`_ 
-   to install everything at once.  
+.. important:: As of version |version|, these instructions presume that your 
+   system already has the library installed to the default location, as outlined 
+   in our :doc:`install` documentation. 
 
 
-#. Set the ``NGRAPH_CPP_BUILD_PATH`` and the ``LD_LIBRARY_PATH`` path to the 
-   location where you built the nGraph libraries. (This example shows the default 
-   location):
+#. Set the ``NGRAPH_CPP_BUILD_PATH`` and the ``LD_LIBRARY_PATH``. You can use 
+   the ``env`` command to see if these paths have been set already and if they 
+   have not, they can be set with something like: 
 
    .. code-block:: bash
 
-      export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist/
-      export LD_LIBRARY_PATH=$HOME/ngraph_dist/lib/       
+      export NGRAPH_CPP_BUILD=path/to/ngraph_dist
```

Comment:
Can we normalize TF and mxnet to use the same environment variable?

Reply:
I'm not sure we even need the CPP build path any more for neon?